app/src/main/java/org/p2p/wallet/auth/interactor/AuthInteractor.kt

```diff
@@ -132,16 +127,7 @@ class AuthInteractor(
         secureStorage.saveString(KEY_PIN_CODE_BIOMETRIC_HASH, currentHash, cipher)
     }
 
-    fun disableBiometricSignIn(untilNextSignIn: Boolean = false) {
-        if (untilNextSignIn) {
-            sharedPreferences.edit { putBoolean(KEY_ENABLE_FINGERPRINT_ON_SIGN_IN, true) }
-        }
-        secureStorage.remove(KEY_PIN_CODE_BIOMETRIC_HASH)
-    }
-
-    fun isAuthorized() = with(sharedPreferences) {
```

Comment:
we were blessed that secureStorage using the same sharedPreferences instead of another one